src/enochecker/results.py

```diff
@@ -67,10 +68,21 @@ def __init__(
         self,
         *args: Any,
         scoreboard_message: Optional[str] = None,
+        internal_message: Optional[str] = None,
         **kwargs: Dict[Any, Any]
     ):
-        super().__init__(*args)
-        self.message = scoreboard_message
+        super().__init__(*args, **kwargs)
+        if scoreboard_message:
+            import warnings
+
+            warnings.warn(
+                "scoreboard_message is deprecated. Use the normal message.",
+                DeprecationWarning,
+            )
+            self.message = scoreboard_message
+        else:
+            self.message = None
```

Comment:
Rather than `None` this needs to be `""`

Reply:
Can't we declare it optional?